server/gitrest/src/utils/helpers.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IGetRefParamsExternal } from "@fluidframework/server-services-client";
+import { IExternalWriterConfig } from "./definitions";
+
+/**
+ * Validates that the input encoding is valid
+ */
+ export function validateBlobEncoding(encoding: BufferEncoding): boolean {
+    return encoding === "utf-8" || encoding === "base64";
```

Comment:
Is it possible that encoding could be the variant `"utf8"`? Does that need to be included here?

Reply:
The type `ICreateBlobParams` from `@fluidframework/gitresources` defines the `encoding` type to be `"utf-8" | "base64"`, so I think this intentionally leaves out `"utf8"`. I'm fairly sure there are no current cases where it could be `"utf8"`, but good lookin' out :)